src/main/java/sentinel/command/Command.java

```diff
@@ -0,0 +1,55 @@
+package sentinel.command;
+
+import sentinel.Sentinel;
+import sentinel.exception.SentinelException;
+import sentinel.ui.Ui;
+import sentinel.utils.SentinelList;
+
+/**
+ * The Command class is an abstract base class for all commands in the Sentinel application.
+ * Each command interacts with the user interface and the task list to perform specific operations.
+ */
+public abstract class Command {
+    protected Ui ui;
+    protected SentinelList list;
```

Comment:
Name could be more descriptive. Maybe something like `taskList` or `userTasks`  :)